src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -56,21 +56,45 @@ class IfTypes(int, Enum):
 class ArpUpdater(MIBUpdater):
     def __init__(self):
         super().__init__()
+        self.db_conn = Namespace.init_namespace_dbs()
         self.arp_dest_map = {}
         self.arp_dest_list = []
         self.arp_dest_map = {}
         self.arp_dest_list = []
+        self.neigh_key_list = {}
 
-    def update_data(self):
-        self.arp_dest_map = {}
-        self.arp_dest_list = []
+    def reinit_data(self):
+        if len(self.db_conn) > 1:
+            Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB)
+            self.neigh_key_list = Namespace.dbs_keys_namespace(self.db_conn, mibs.APPL_DB, "NEIGH_TABLE:*")
+
+    def _update_host_data(self):
         for entry in python_arptable.get_arp_table():
             dev = entry['Device']
             mac = entry['HW address']
             ip = entry['IP address']
-
+            self._update_arp_info(dev, mac, ip)
+
+    def _update_namespace_data(self):
+        for neigh_key in self.neigh_key_list:
+            neigh_str = neigh_key.decode()
+            db_index = self.neigh_key_list[neigh_key]
+            dev, ip = mibs.get_neigh_info(neigh_str)
+            """
```

Comment:
>""" [](start = 12, length = 3)

Docstrings are for all public modules, functions, classes, and methods. Here '# ' will be better.

Reply:
Fixed.